.gitignore

```diff
@@ -44,22 +45,25 @@ ios/ParentBuddyApp.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks
 ios/ParentBuddyApp.xcodeproj/xcuserdata/kalyanikaushik.xcuserdatad/xcschemes/xcschememanagement.plist
 ios/GoogleServices/Xk/GoogleService-Info_old.plist
 ios/GoogleServices/Production/GoogleService-Info_new.plist
+ios/GoogleServices/MerhabaBebek/GoogleService-Info.plist
 ios/ParentBuddyApp/Info.plist
 ios/ParentBuddyAppXk-Info.plist
 ios/ParentBuddyApp/Info.plist
 ios/ParentBuddyAppXk-Info.plist
+ios/ParentBuddyAppBangla-Info.plist
+ios/ParentBuddyAppBanglaDev-Info.plist
 ios/GoogleServices/Development/GoogleService-Info.plist
```

Comment:
As per our naming convention these folders should be:
bebboDevelopment instead of Development
bebbo instead of Production
babuni instead of Bangla
babuniDevelopment instead of BanglaDevelopment
foleja instead of Xk
folejaDevelopment instead of XkDevelopment